.travis.yml

```diff
@@ -4,21 +4,30 @@ node_js:
   - '4'
   - 'stable'
 addons:
+  firefox: 'latest-beta'
   apt:
     sources:
     - ubuntu-toolchain-r-test
+    - google-chrome
     packages:
     - g++-4.8
+    - google-chrome-stable
```

Comment:
Nit: We could use the `chromium-browser` package so we don't have to add the `google-chrome` PPA earlier.